zq/core/config.py

```diff
@@ -0,0 +1,8 @@
+
+SECRET_KEY = "a77ec4c45a3bf81f68882a5f2398c26e953d16dc5c3c63c34730f23b19654e31"
```

Comment:
please keep this in DB, first time bootstrap should generate and store it in DB if not exist.